README.md

```diff
@@ -262,6 +262,28 @@ brew install pipenv
 CFLAGS=“-I$(brew --prefix openssl)/include -I$(brew --prefix bzip2)/include -I$(brew --prefix readline)/include -I$(xcrun --show-sdk-path)/usr/include” LDFLAGS=“-L$(brew --prefix openssl)/lib -L$(brew --prefix readline)/lib -L$(brew --prefix zlib)/lib -L$(brew --prefix bzip2)/lib” pyenv install --patch 3.6.8 < <(curl -sSL https://github.com/python/cpython/commit/8ea6353.patch\?full_index\=1)
 ```
 
+Monterey (M1) Addendum
+-----------------------
+
+Since Apple's new M1 chip release, there has been a considerable number of processor-specific errors in regard to Python dependencies that are documented in upstream vendor bug tickets through GitHub. 
+
+The issues that have been experienced so far have revolved around installing the `pienv` tool. The following workaround can be implemented after running the `laptop` script (as of March 2022) that we recommend:
+
+- **AirPlay Receiver:** Navigate to your MacOS `System Preferences`, then select `Sharing`. If the `Airplay Receiver` radio button is selected, make sure to unselect it and ensure you are set to  `AirPlay Receiver: Off`. <-- something about the port used? need clarification please :) -->
+
+- **Rosetta Terminal:** 
+ - Navigate to your MacOS `Finder`, then select your `Applications` folder, then `Utilities`, then right-click your Terminal app and select `Duplicate`.
+ - Rename the newly-duplicated Terminal to an easy-to-remember label (ex: rosetta-terminal).
+ - Right-click the `rosetta-terminal` and click on `Get Info`.
+ - Check the option for **Open using Rosetta**. 
+ - This emulates this terminal environment to i386 (you can verify this by opening `rosetta-terminal`, and running the `arch` command. This should verify the processor emulation is running i386. Your iTerm2 application and native terminal should still show arm64 using the same `arch` command.
+
+- **Manually configure NPM_TOKEN**
+ - Navigate to your `.zshrc` file and open using a code editor/IDE.
+ - append the file by adding in `export NPM_TOKEN="_insert NPM token here as a string value_"`.
+
+**WORK IN PROGRESS**
```

Comment:
Would love to see two more sections here with code examples in Markdown formatting (see elsewhere in README.md).

(1) Let's describe the steps to install the NVM tool and the Node.js toolchain at version 10.15.3 (per the `mac` script in this repo for non-M1 users). A description of using that Rosetta Terminal instance and a few snippets of code, I think - gets us there?

(2) For the Python toolchain - I think we could describe the non-Rosetta Terminal workflow that solved our problems. Basically - we (a) installed the pre-built executable from https://www.python.org/downloads/release/python-368/ and then (b) we then ran a `brew install pipenv`.

For the remaining steps - those specific to Mariana Django - let's consider maybe updating the Quickstart Guide and then linking that doc here? Maybe something like `For those who may be continuing on to run the Mariana Django project (as of March 2022), see M1 specific dependency instructions [here]`.

That secondary bit of documentation really just includes (1) describing the setting of the shell variables (see screenshot), and (2) running `pipenv --python /usr/local/bin/python3 install --dev` only after having set those.

![Screenshot from 2022-03-28 16-37-03](https://user-images.githubusercontent.com/10262117/160498790-8ca2450a-3888-407e-997d-8a531c962707.png)